src/main/java/com/google/api/codegen/viewmodel/PathTemplateCheckView.java

```diff
@@ -23,6 +23,8 @@
 
   public abstract String paramName();
 
+  public abstract String methodName();
```

Comment:
Should this be called something that indicates its function, rather than its content? validationMessagePrefix, or similar?